.markdownlint.yaml

```diff
@@ -3,3 +3,6 @@
   "code_blocks": false # Disable rule for hard tabs in code blocks
 "MD013": false # Disable rule for line length
 "MD033": false # Disable rule banning inline HTML
+"MD025": false # Disable rule for multiple top-level headers
```

Comment:
[no change needed] I like this rule. Are we violating a lot (hence why we ignore it here)?

Reply:
good callout, i'll add it just to the SUMMARY.md file for the sidebar structure

Reply:
https://github.com/celestiaorg/CIPs/pull/170/commits/b093bb02136fa161eec9a6f7b6f447396e149840